acra/acra-in-depth/architecture/_index.md

```diff
@@ -1,4 +1,37 @@
 ---
 title: Architecture
 bookCollapseSection: true
+weight: 1
 ---
+
```

Comment:
```suggestion
**TODO: Expand**
```
Will be great to have some marker that we can `grep` and find where we should do something.

Reply:
This is indeed intended marker for grep,- I did grep the whole documentation repo before starting to use it and I think TODO (ALLCAPS) is not used anywhere for reasons other than actual todo. 

Moreover, I believe, **TODO: Expand** is unique enough to grep for it.

Reply:
I suggested adding this line here because it was missed here and in the `transport-security/_index.md` files. and exists in all other docs that you updated. Yes, we used TODO as marker. And because this doc file is not finished, it will be good to add `TODO` line too